src/views/index_view.js

```diff
@@ -31,6 +31,16 @@ export default BaseView.extend({
     // up arrow and shift+tab move up
     } else if (data.key === 'ArrowUp' || (data.key === 'Tab' && data.shiftKey)) {
       this._selectNextItem(-1);
+    // enter triggers a navigation
+    } else if (data.key === 'Enter') {
+      let selectedItem = this.query('li.selected');
+      if (selectedItem) {
+        // trick the element into navigating
+        // TODO: use JS events, not DOM events, for signaling
+        var fakeClick = new CustomEvent('mousedown');
+        fakeClick.which = 1;
```

Comment:
Yeah this seems fine for now. What if we move [this](https://github.com/mozilla/universal-search-content/blob/handle-enter-on-frame/src/views/index_view.js#L36-L42) into its own function and create an issue to remind us to replace it?